darkmoon/api/v1/metadata/Exception_Response.py

```diff
@@ -0,0 +1,84 @@
+class Missing_Hash_Exception(Exception):
```

Comment:
Please remove underscores from all class names in accordance with[PEP8](https://peps.python.org/pep-0008/#class-names)